### ISSUE TITLE
Error running reproduce.sh , no attribute `reap_state`

### ISSUE THREAD

**Issue by sbenthall** [econ-ark/KrusellSmith#4]
Getting this error running `reproduce.sh`:

https://gist.github.com/sbenthall/419ae456a640a8a2de7a88145856ff1e

**Comment by MridulS** [comment]
yeah it should be econ-ark v0.10.8 not 0.10.7, updated the environment.yml file which also pins the python version.

**Comment by MridulS** [comment]
https://github.com/econ-ark/KrusellSmith/commit/353c7315c8c8d0f5728c5ec2c27a891bd09b8f3f removed the requirements.txt file.

**Comment by sbenthall** [comment]
it looks like `reproduce.sh` is supposed to create and activate a conda environment with the dependencies installed.

It does not work on my system.

So there are additional setup requirements not yet documented to getting `reproduce.sh` to work.

I have not tried the README instructions, which refer to stuff that is not in the REMARK standard.

Also, this repo has to release tag, and no version is specified in the metadata in the REMARK repository.
